src/main/java/nl/rijksoverheid/mev/gezagsmodule/service/ToelichtingService.java

```diff
@@ -0,0 +1,81 @@
+    private static final String FORMAT_SIGN = "%s";
+    private static final String SPACE = " ";
+    private static final String DOT_ENDLINE = ".\n";
+
+    /**
+     * Past de basis toelichting zoals opgenomen in het antwoordenmodel aan op basis van of velden in onderzoek of missende gegevens.
+     *
+     * @param baseToelichting   de basis toelichting
+     * @param veldenInOnderzoek de velden die in onderzoek waren - of null
+     * @param missendeGegevens  de missende gegevens - of null
+     * @return de toelichting zoals uit het antwoorden model bewerkt met de additionele gegevens
+     */
+    public String decorateToelichting(final String baseToelichting, final VeldenInOnderzoek veldenInOnderzoek, final List<String> missendeGegevens) {
+        StringBuilder sb = new StringBuilder();
+        if (baseToelichting != null && !baseToelichting.isEmpty()) {
+            if (veldenInOnderzoek != null && veldenInOnderzoek.hasValues()) {
+                sb.append(baseToelichting);
+                sb.append(SPACE);
+                setInOnderzoek(sb, veldenInOnderzoek);
+            } else if (missendeGegevens != null && !missendeGegevens.isEmpty()) {
+                setMissendeGegevens(sb, baseToelichting, missendeGegevens);
+            }
+        }
+        
+        return sb.toString();
+    }
+
+    private void setInOnderzoek(final StringBuilder sb, final VeldenInOnderzoek veldenInOnderzoek) {
+        sb.append(IN_ONDERZOEK);
+        List<String> persoonInOnderzoekVelden = veldenInOnderzoek.getPersoon();
+        if (!persoonInOnderzoekVelden.isEmpty()) {
+            sb.append(IN_ONDERZOEK_PERSOONSVELDEN);
+            sb.append(String.join(", ", persoonInOnderzoekVelden));
+            sb.append(DOT_ENDLINE);
+        }
+        List<String> ouder1InOnderzoekVelden = veldenInOnderzoek.getOuder1();
+        if (!ouder1InOnderzoekVelden.isEmpty()) {
+            sb.append(IN_ONDERZOEK_VELDEN_OUDER_1);
+            sb.append(String.join(", ", ouder1InOnderzoekVelden));
+            sb.append(DOT_ENDLINE);
+        }
+        List<String> ouder2InOnderzoekVelden = veldenInOnderzoek.getOuder2();
+        if (!ouder2InOnderzoekVelden.isEmpty()) {
+            sb.append(IN_ONDERZOEK_VELDEN_OUDER_2);
+            sb.append(String.join(", ", ouder2InOnderzoekVelden));
+            sb.append(DOT_ENDLINE);
+        }
+        List<String> nietOuderInOnderzoekVelden = veldenInOnderzoek.getNietOuder();
+        if (!nietOuderInOnderzoekVelden.isEmpty()) {
+            sb.append(IN_ONDERZOEK_VELDEN_NIET_OUDER);
+            sb.append(String.join(", ", nietOuderInOnderzoekVelden));
+            sb.append(DOT_ENDLINE);
+        }
+    }
+
+    private void setMissendeGegevens(final StringBuilder sb, final String baseUitleg, final List<String> missendeGegevens) {
+        if (baseUitleg.contains(FORMAT_SIGN)) {
+            sb.append(String.format(baseUitleg, String.join(", ", missendeGegevens)));
```

Comment:
`baseUitleg.contains(PLACEHOLDER)` dekt de lading beter denk ik.

Ook kan je tegenwoordig gelijk `.format` aanroepen op een string, dus 

```java
if (baseUitleg.contains(PLACEHOLDER)) {
    baseUitleg.format(String.join(", ", missendeGegevens))
}
```

Dat is iets korter.